romanisim/gaia.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+from astropy import table, coordinates, units as u
+
+
+def gaia2romanisimcat(gaiacat, date, refepoch=2016.0, boost_parallax=1,
+                      fluxfields=['F184']):
+    """Convert Gaia output to a catalog of locations for input to romanisim.
+
+    Mostly an astrometry routine, doing the usual parallax and proper motion
+    computations by hand.  Caveat emptor---may all be wrong!
+
+    boost_parallax is intended to allow accounting for the fact that parallaxes
+    are for 1 AU, but Roman will be at 1.01 AU (L2) and so everything moves by
+    1% more than you might expect.  If one wants to do better than that one
+    needs to think about the detailed orbit of Roman in L2; the location
+    of L2 itselfs moves annually and monthly due to the eccentricity of the
+    Earth's orbit and the fact that the Earth is not at the Earth-moon
+    barycenter.
+
+    Parameters
+    ----------
+    gaiacat : astropy.table.Table
+        Gaia catalog, containing at least ra, dec, pmra, pmdec, parallax,
+        phot_g_mean_mag fields
+    date : astropy.time.Time
+        Time of observation
+    refepoch : float
+        Reference epoch of Gaia parameters
+    boost_parallax : float
+        Amount to boost parallaxes by (L2 describes a wider orbit than the
+        Earth's)
+    fluxfields : list[str]
+        List of strings to fill with estimated fluxes
+
+    Returns
+    -------
+    catalog : astropy.table.Table
+        astropy Table formatted for input to romanisim
+    """
+    outcat = table.Table()
+    dt = date.jyear - refepoch
```

Comment:
For the sake of consistency, I think we should define `refepoch` as a `Time` object (like `date`) and do `dt = date.jyear - refepoch.jyear`.

Reply:
I thought about that.  But I don't think as a user we ever really want to have more flexibility than accepting Julian years for this routine.  If I had the more general Time interface, that would let people use whatever random times they have floating around, which I guess would be nice.  But it would also mean that when Gaia says their reference epoch is 2016.0, I'd need to read about how to transform 2016.0 into an astropy Time object, rather than just entering 2016 here.  I was happy to see that Time(..., format='jyear') was a thing, but I'm happy to save future users needing to relearn that.